.github/workflows/prebuild.yml

```diff
@@ -0,0 +1,33 @@
+name: core-prebuild
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
maybe we should build only on "offical" release
```yaml
  tags:
    - "v*"
```

and maybe ignore prerelease tags if that's possible (probably a `- "!*alpha"` or somehting like that, need to dig more the doc)